Packs/CheckPointSandBlast/Integrations/CheckPointSandBlast/CheckPointSandBlast.yml

```diff
@@ -21,6 +21,20 @@ configuration:
   name: proxy
   required: false
   type: 8
+- name: integrationReliability
+  additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: C - Fairly reliable
+  display: Source Reliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: false
+  type: 15
```

Comment:
Important addition, but you have not used it in the python code.

Reply:
https://github.com/demisto/content-docs/pull/1205

Reply:
You are right, it was hardcoded as `C - Fairly reliable`, changed it to refer to the integration parameter value while its default is  `C - Fairly reliable`